tests/runtime/call

```diff
@@ -477,23 +477,13 @@ EXPECT 123
 TIMEOUT 1
 
 NAME print_avg_map_args
-PROG BEGIN { print("BEGIN"); @["a"] = avg(10); @["b"] = avg(20); @["c"] = avg(30); @["d"] = avg(40); print(@, 2, 10); print("END"); clear(@); exit(); }
-EXPECT BEGIN
-       @[c]: 3
-       @[d]: 4
-       
-       END
+PROG BEGIN { @["a"] = avg(10); @["b"] = avg(20); @["c"] = avg(30); @["d"] = avg(40); print(@, 2, 10); clear(@); exit(); }
+EXPECT_FILE  runtime/outputs/print_avg_map_args.txt
 TIMEOUT 1
 
 NAME print_avg_map_with_large_top
```

Comment:
ugh both of these tests could use better/more-descriptive names

Reply:
Ideas welcome :P

Reply:
Ha, I'll put up a PR if I think of something.